packages/keyvAdapter/README.md

```diff
@@ -18,9 +18,10 @@ In Apollo Server v3:
 import { ApolloServer } from "apollo-server";
```

Comment:
Could you add something to the README explaining that (if I understand correctly) v4 of this package works with keyv v5, and you can use v3 of this package for keyv v4?

Reply:
Yes, I added a note about it.